packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx

```diff
@@ -117,3 +129,29 @@ export const useLocalVideoStartTrigger = (isLocalVideoAvailable: boolean, should
     }
   }, [shouldTransition, isButtonStatusSynced, isPreviewCameraOn, isLocalVideoAvailable, mediaGalleryHandlers]);
 };
+
+/**
+ * Hook to fetch new participant avatar and video gallery displayName information if it is present.
+ * @param onFetchAvatarPersonaData
+ */
+const useOnFetchAvatarPersonaData = (
```

Comment:
Rename to better reflect what this does: `useRemoteParticipantsWithCustomDisplayNames` (or something shorter if you can think one up).